pkg/status/internal/pb/status.proto

```diff
@@ -15,9 +15,10 @@ message Get {}
 // the appropriate values that are a snapshot of the current state
 // of the running node.
 message Snapshot {
-  uint64 ReserveSize = 1;
-  double PullsyncRate = 2;
-  uint32 StorageRadius = 3;
-  uint64 ConnectedPeers = 4;
-  uint64 NeighborhoodSize = 5;
+  string BeeMode = 1;
```

Comment:
If we want to maintain backward compatibility, we should be adding this field at the end I think. This allows us to handle messages from older protocol as well.

Reply:
So the minor version of the protocol is bumped to cover that.